src/components/ui/image.jsx

```diff
@@ -30,6 +30,15 @@ let Image = React.createClass({
     }
   },
 
+  componentWillReceiveProps(nextProps) {
+    let { src } = this.state
+
+    this.setState({
+      isLoaded: nextProps.src !== src,
+      src: nextProps.src
+    });
+  },
```

Comment:
Ah okay. Yay, tests! 

-- 
Nate Hunzaker

On October 19, 2015 at 3:10:49 PM, Chris Manning (notifications@github.com) wrote:

In src/components/ui/image.jsx:

> @@ -30,6 +30,15 @@ let Image = React.createClass({
>      }
>    },
>   
> +  componentWillReceiveProps(nextProps) {
> +    let { src } = this.state
> +
> +    this.setState({
> +      isLoaded: nextProps.src !== src,
> +      src: nextProps.src
> +    });
> +  },
I was using state instead of the src

—
Reply to this email directly or view it on GitHub.